src/coreclr/tools/superpmi/mcs/verbmerge.cpp

```diff
@@ -39,6 +46,15 @@ char* verbMerge::ConvertWideCharToMultiByte(LPCWSTR wstr)
     return encodedStr;
 }
 
+WCHAR* verbMerge::ConvertMultiByteToWideChar(LPCSTR str)
+{
+    unsigned int codePage   = CP_UTF8;
+    int          sizeNeeded = MultiByteToWideChar(codePage, 0, str, -1, NULL, 0);
+    WCHAR*       encodedStr = new WCHAR[sizeNeeded];
+    MultiByteToWideChar(codePage, 0, str, -1, encodedStr, sizeNeeded);
```

Comment:
As follow-up, could we convert these two methods (ConvertMultiByteToWideChar and ConvertWideCharToMultiByte) to use the minipal as they're both UTF8-only? It would allow us to further reduce the usage of the CoreCLR PAL within superpmi.

Reply:
Makes sense